packages/webservice/models/src/main/kotlin/com/yahoo/navi/ws/models/beans/Role.kt

```diff
@@ -0,0 +1,47 @@
+/**
+ * Copyright 2020, Yahoo Holdings Inc.
+ * Licensed under the terms of the MIT license. See accompanying LICENSE.md file for terms.
+ */
+package com.yahoo.navi.ws.models.beans
+
+import com.yahoo.elide.annotation.Include
+import com.yahoo.elide.annotation.SharePermission
+import com.yahoo.elide.annotation.DeletePermission
+import com.yahoo.elide.annotation.CreatePermission
+import com.yahoo.elide.annotation.UpdatePermission
+import org.hibernate.annotations.CreationTimestamp
+
+import java.util.Date
+
+import javax.persistence.Entity
+import javax.persistence.Table
+import javax.persistence.Id
+import javax.persistence.Column
+import javax.persistence.Temporal
+import javax.persistence.TemporalType
+import javax.validation.constraints.NotBlank
+
+@Entity
+@Table(name = "navi_roles")
+@Include(rootLevel = true, type = "roles")
+@SharePermission
+@DeletePermission(expression = "everybody")
+@CreatePermission(expression = "everybody")
+@UpdatePermission(expression = "nobody")
+class Role {
+    @get:Id
+    @get:NotBlank
+    var id: String? = null
+
+    @get:CreationTimestamp
+    @get:Column(columnDefinition = "timestamp default current_timestamp")
+    @get:Temporal(TemporalType.TIMESTAMP)
+    @get:UpdatePermission(expression = "nobody")
+    var createdOn: Date? = null
+
+    @get:CreationTimestamp
```

Comment:
Should this be `UpdateTimestamp`?
https://docs.jboss.org/hibernate/orm/4.3/javadocs/org/hibernate/annotations/UpdateTimestamp.html